static/app/views/settings/organization/navigationConfiguration.tsx

```diff
@@ -58,7 +58,7 @@ const organizationNavigation: NavigationSection[] = [
       {
         path: `${pathPrefix}/audit-log/`,
         title: t('Audit Log'),
-        show: ({access}) => access!.has('org:write'),
+        show: ({access}) => access!.has('org:read'),
```

Comment:
i think we can remove this if we always want to show it. we do have a role that doesn't have `org:read` in SaaS